src/lib.rs

```diff
@@ -174,9 +174,9 @@ impl NixConfig {
 pub enum ParseError {
     #[error("file '{0}' not found")]
     FileNotFound(PathBuf),
-    #[error("file '{0}' included from '{}' not found", .1.as_ref().map(|path| path.display().to_string()).unwrap_or(String::from("<unknown>")))]
+    #[error("file '{0}' included from '{origination}' not found", origination=.1.as_ref().map(|path| path.display().to_string()).unwrap_or(String::from("<unknown>")))]
```

Comment:
nit: `origination` feels kind of clunky. Any reason we couldn't pick a simpler name like below?

```suggestion
    #[error("file '{0}' included from '{file}' not found", file=.1.as_ref().map(|path| path.display().to_string()).unwrap_or(String::from("<unknown>")))]
```